e2e/_suites/helm/features/apm-server.feature

```diff
@@ -1,3 +1,4 @@
+@apm-server
```

Comment:
Bring back Cucumber tags, so we will simply filter test execution by tags. As an example, for the metricbeat test suite: `godog -t redis`